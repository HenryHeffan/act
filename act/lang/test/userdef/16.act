template<pint N>
deftype foo <: int<1> (bool a[N], b[N])
{
  a = b;
}

deftype bar <: foo<5> ( ) 
{ 
}



defproc testme (foo <5> f)
{
bar b;

  f = b;
}

testme t;
