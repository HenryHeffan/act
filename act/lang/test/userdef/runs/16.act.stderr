ERROR: File `16.act', line 17, col 5
	Typechecking failed on connection!
	Connection is compatible, but requires non-local instance (LHS) to be specialized
